nemo/collections/asr/models/aed_multitask_models.py

```diff
@@ -783,17 +783,6 @@ def _transcribe_on_begin(self, audio, trcfg: MultiTaskTranscriptionConfig):
                     trcfg._internal.primary_language = self.tokenizer.langs[0]
                     logging.debug(f"Transcribing with default setting of {trcfg._internal.primary_language}.")
 
-        elif isinstance(audio, str):
```

Comment:
Why are we removing manifest ability from multi task model ?

Reply:
its not removing but same check is now already part of TranscriptionMixin to handle manifest